lips/lip-10.md

```diff
@@ -0,0 +1,221 @@
+   "currency": "<currency>"
+}
+```
+
+| Field 	   | Type 	     | Required? 	| Description 	           |
+|-------	   |------	     |-----------	|-------------	           |
+|sender        |[User Object](https://github.com/libra/lip/blob/master/lips/lip-10.md#user-object)  | Y            | Information about the sender |
+|receiver_id   |str          | Y            | Libra ID of the receiver |
+|amount        |integer      | Y            | Amount of coins to transfer |
+|currency      |str          | Y            | Name of the coin |
+
+**Response:**
+```json
+{
+   "receiver_address": "<Receiver VASP on-chain account address>",
+   "reference_id": "<Payment_Reference_ID>",
+   "receiver": "<User Object>"
+}
+```
+
+| Field 	      | Type       | Required? 	| Description 	           |
+|-------	      |------      |-----------	|-------------	           |
+|receiver_address |str         | Y            | On-chain account address for receiving the payment. This address must be encoded as a [bench 32 account identifier](https://github.com/libra/lip/blob/master/lips/lip-5.md#account-identifiers) **without** subaddress |
+|reference_id     |str         | Y            | Reference ID to be included into payment metadata |
+|receiver         |[User Object](https://github.com/libra/lip/blob/master/lips/lip-10.md#user-object) | Y            | Information about the receiver |
+
+* Pay endpoint is used to exchange details of the payment such as information about Sender and Receiver off the chain, and negotiate `Payment_Reference_ID`
+
+* Once the sender learns Payment_Reference_ID, it will submit an on-chain transaction including the `Payment_Reference_ID` as a metadata. Receiver will use the `Payment_Reference_ID` in the transaction to link the on-chain transaction to the payment.
+
+* Sender will submit a transaction to the `receiver_address` on-chain account in the response. This account address can be either Parent VASP, or any of Child VASP accounts of the receiver VASP. This is needed to allow the receiver VASP to manage load between different on-chain accounts.
+
+* `Payment_Reference_ID` must be unique for each payment, it must not be reused in any way to avoid leaking privacy information on-chain.
+
+* `Payment_Reference_ID` returned by this endpoint is not signed for KYC purposes. This means that payment above KYC threshold can not be submitted right away using the provided Payment_Reference_ID. Instead, sender VASP needs to proceed to KYC endpoint to negotiate the signed Reference ID.
+
+* KYC endpoint API will be updated to accept `Payment_Reference_ID` returned by the Libra ID endpoint instead of address/subaddress.
+
+## User Object
+User Object defines reach information about the user.
+
+All fields, except for `libra_id`, in this object are optional - VASP can choose to share either of those fields or none.
+
+Additionally, all fields in this object are informational - **they must not be used for KYC and only intended for UI purposes**.
```

Comment:
rewrite are information, that is they are intended for UI purposes only and explicitly not establishing KYC.